src/util.ts

```diff
@@ -20,3 +20,23 @@ export const toNumber = (value: number | bigint) => {
 }
 
 export class UnsafeConversionError extends Error {}
+
+/**
+ * Calculates the amount of funds to add to a wallet to bring it up to the minimum spending balance.
+ * @param minSpendingBalance The minimum spending balance for the wallet
+ * @param currentSpendingBalance The current spending balance for the wallet
+ * @param minFundingIncrement The minimum amount of funds that can be added to the wallet
+ * @returns The amount of funds to add to the wallet or null if the wallet is already above the minimum spending balance
+ */
+export const calculateFundAmount = (
```

Comment:
Is this exported into algokit itself? If not then all good, if it is we probably need a better name for the function that's more descriptive.

Reply:
@robdmoore thats all thats exported, don't see utils here so should be good:

```ts
import { UpdatableConfig } from './types/config'

export * from './account'
export * from './amount'
export * from './app'
export * from './app-client'
export * from './app-deploy'
export * from './indexer-lookup'
export * from './localnet'
export * from './network-client'
export * from './transaction'
export * from './transfer'
export * from './dispenser-client'

/** The AlgoKit config. To update it use the configure method. */
export const Config = new UpdatableConfig()
```